src/Servers/Kestrel/Core/src/KestrelServerOptions.cs

```diff
@@ -161,6 +161,21 @@ public class KestrelServerOptions
         /// </summary>
         internal bool IsDevCertLoaded { get; set; }
 
+        private bool? _enableLineFeedTerminator;
+        internal bool EnableLineFeedTerminator
+        {
+            get
+            {
+                if (!_enableLineFeedTerminator.HasValue)
+                {
+                    _enableLineFeedTerminator = AppContext.TryGetSwitch("Microsoft.AspNetCore.Server.Kestrel.EnableLineFeedTerminator", out var enabled) && enabled;
+                }
+                
+                return _enableLineFeedTerminator.Value;
+            }
```

Comment:
For reference, when porting to main this property would become public and the AppContext switch removed.